content/user/ui/settings/Integrations.md

```diff
@@ -16,12 +19,12 @@ To add, go to your team's **Build** settings in Slack:
 * Choose the appropriate channel to send messages to
```

Comment:
Should we say "Choose the appropriate channel to which you'd like to send messages" (following syntax used elsewhere in the article)?